backend/app/src/test/kotlin/io/tolgee/batch/BatchJobOperationTimer.kt

```
@@ -0,0 +1,115 @@
+    logger.info("BATCH JOB OPERATION TIMING REPORT")
+    logger.info("=".repeat(100))
+    logger.info(
+      String.format(
+        "%-50s %10s %12s %12s %12s %12s",
+        "Operation",
+        "Count",
+        "Total(ms)",
+        "Avg(ms)",
+        "Min(ms)",
+        "Max(ms)",
+      ),
```

detekt (1.23.8)

[warning] String.format(
        "%-50s %10s %12s %12s %12s %12s",
        "Operation",
        "Count",
        "Total(ms)",
        "Avg(ms)",
        "Min(ms)",
        "Max(ms)",
      ) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)